Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:8.12-alpine
+
+ADD . /app
+WORKDIR /app
+
+ARG COLLECT_USAGE_DATA=true
```

Comment:
I think this should probably be false by default

Reply:
When pushing an image to the platform, interactive mod isn't a thing. Nor is by default on local environments.

We considered setting this to `false`, however we didn't want to step on your toes too much :)

Happy to set it to `false`.